views/documentation.pug

```diff
@@ -0,0 +1,18 @@
+extends layout.pug
+
+block head
+	.navbar
```

Comment:
Your approach has a lot of redundant code maybe consider using a partial or better yet make this the default and override in index